### ISSUE TITLE
Error Panel only flashs across when the PDF fails to compile in the first time

### ISSUE THREAD

**Issue by Honghe** [jupyterlab/jupyterlab-latex#80]
Environment:

```
(jupyter_py3.6) ➜  jupyter jupyter --version          
4.4.0
(jupyter_py3.6) ➜  jupyter jupyter lab --version      
0.32.1
(jupyter_py3.6) ➜  jupyter jupyter labextension list  
JupyterLab v0.32.1
Known labextensions:
   app dir: /home/honhe/.pyenv/versions/3.6.5/envs/jupyter_py3.6/share/jupyter/lab
@jupyterlab/latex
        @jupyterlab/latex v0.3.1  enabled  OK
```

**Comment by ian-r-rose** [comment]
Thanks for the report @Honghe! I also see this for the first failed compile (though if I trigger another save, it shows up correctly).

**Comment by macbuse** [comment]
I had the same problem with the same jupyter configuration as OP.

Doing a save didn't resolve the issue for me whilst trying to compile 

https://github.com/jupyterlab/jupyterlab-latex/blob/master/sample.tex

The problem seemed to be that the pdf didn't exist and I was getting an error
404 GET /api/contents/sample .pdf?format

----------------------------------------------------------------------------------------------
Solution : 

- deleted  the \includegraphics, did pdflatex in CLI to generate pdf

- tried again in jupyter and got the panel with a long python error
and  got a preview (because I used pdflatex in the terminal) .

- generated  jupyter_notebook_config.py file and added 
c.LatexConfig.latex_command = 'pdflatex'

- works fine without   \includegraphics  but  the panel still flashes  on the 
original sample.tex

------------------------------------------------------

In any case this is **amazing! I** didn't expect this to work at all as my installation is a little "exotic" :

Asus chromebook c301
Chrome OS Version 68.0.3440.70 (Official Build) beta (64-bit) - developer mode

Anaconda 5.2 from https://www.anaconda.com/download/#linux
TexLive quickinstall from https://www.tug.org/texlive/quickinstall.html

**Comment by ian-r-rose** [comment]
Fixed by #95.
